tiledb/sm/query/sparse_unordered_with_dups_reader.cc

```diff
@@ -454,70 +460,79 @@ Status SparseUnorderedWithDupsReader::create_result_tiles(bool* tiles_found) {
     bool budget_exceeded = false;
     unsigned int f = 0;
     while (f < fragment_num && !budget_exceeded) {
-      auto range_it = result_tile_ranges_[f].rbegin();
-      uint64_t t = 0;
-      while (range_it != result_tile_ranges_[f].rend()) {
-        for (t = range_it->first; t <= range_it->second; t++) {
+      if (!all_tiles_loaded_[f]) {
```

Comment:
Is 
A)this set somewhere else than below [old line 487] 
or 
B) now irrelevant,
since its only showing as no longer being set and no new location for it being set (in diffs).
Similar observation in 'else' branch below...

Reply:
It's now set in SparseUnorderedWithDupsReader::add_result_tile, line 393.